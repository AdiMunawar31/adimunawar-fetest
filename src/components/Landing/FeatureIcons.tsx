const FeatureIcons = () => {
  return (
    <div className='grid grid-cols-3 space-x-4 md:space-x-6 md:flex md:justify-center lg:justify-start'>
      <a
        aria-label='add to slack'
        href='#'
        className='p-4 border border-gray-200 dark:bg-gray-800 dark:border-gray-700 rounded-full duration-300 hover:border-cyan-400 hover:shadow-lg hover:shadow-cyan-600/20 dark:hover:border-cyan-300/30'
      >
        <div className='flex justify-center space-x-4'>
          <img
            className='w-6 h-6'
            src='https://tailus.io/sources/blocks/tech-startup/preview/images/slack.png'
            alt='slack logo'
            loading='lazy'
            width={128}
            height={128}
          />
          <span className='hidden font-medium md:block dark:text-white'>
            Slack
          </span>
        </div>
      </a>
      <a
        aria-label='add to chat'
        href='#'
        className='p-4 border border-gray-200 dark:bg-gray-800 dark:border-gray-700 rounded-full duration-300 hover:border-green-400 hover:shadow-lg hover:shadow-lime-600/20 dark:hover:border-green-300/30'
      >
        <div className='flex justify-center space-x-4'>
          <img
            className='w-6 h-6'
            src='https://tailus.io/sources/blocks/tech-startup/preview/images/chat.png'
            alt='chat logo'
            loading='lazy'
            width={128}
            height={128}
          />
          <span className='hidden font-medium md:block dark:text-white'>
            Google Chat
          </span>
        </div>
      </a>
      <a
        aria-label='add to zoom'
        href='#'
        className='p-4 border border-gray-200 dark:bg-gray-800 dark:border-gray-700 rounded-full duration-300 hover:border-blue-400 hover:shadow-lg hover:shadow-blue-600/20 dark:hover:border-blue-300/30'
      >
        <div className='flex justify-center space-x-4'>
          <img
            className='w-6 h-6'
            src='https://tailus.io/sources/blocks/tech-startup/preview/images/zoom.png'
            alt='chat logo'
            loading='lazy'
            width={128}
            height={128}
          />
          <span className='hidden font-medium md:block dark:text-white'>
            Zoom
          </span>
        </div>
      </a>
    </div>
  );
};

export default FeatureIcons;
