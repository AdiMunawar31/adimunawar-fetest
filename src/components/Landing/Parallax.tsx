import React from "react";

const Parallax = () => {
  return (
    <section
      className='w-full h-[500px] bg-cover bg-fixed bg-center flex flex-col justify-center items-center'
      style={{
        backgroundImage:
          "url(https://colibriwp.com/blog/wp-content/uploads/2022/02/15.jpg)",
      }}
    >
      <div className='text-gray-900 text-3xl sm:text-4xl font-semibold text-center mt-40 sm:mt-24 mb-6'>
        Stay Up-To-Date
      </div>
      <div className='text-center font-bold sm:mt-10 mb-40 sm:mb-28 text-gray-900 max-w-2xl mx-auto p-4'>
        "Empowering you with real-time insights, our app is designed to keep
        your contact information up-to-date. Our automated scans tirelessly work
        behind the scenes, ensuring that you're always equipped with the latest
        and most accurate details. Say goodbye to the frustration of outdated
        contacts and hello to a world where staying informed is effortless. With
        us, you'll never miss a beat, and you'll always be in control of the
        most current information. Welcome to a future of seamless,
        always-accurate connections."
      </div>
    </section>
  );
};

export default Parallax;
