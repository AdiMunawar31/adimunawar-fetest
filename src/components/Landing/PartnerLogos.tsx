import React from "react";

const PartnerLogos = () => {
  return (
    <>
      <div className='text-gray-600 font-bold'>
        <span>Our Partnership :</span>
      </div>
      <div className='flex gap-6 lg:gap-12 justify-between grayscale lg:w-2/3'>
        <img
          src='https://tailus.io/sources/blocks/tech-startup/preview/images/clients/airbnb.svg'
          className='h-8 sm:h-10 w-auto lg:h-12'
          alt='airbnb'
        />
        <img
          src='https://tailus.io/sources/blocks/tech-startup/preview/images/clients/ge.svg'
          className='h-8 sm:h-10 w-auto lg:h-12'
          alt='ge'
        />
        <img
          src='https://tailus.io/sources/blocks/tech-startup/preview/images/clients/coty.svg'
          className='h-8 sm:h-10 w-auto lg:h-12'
          alt='coty'
        />
        <img
          src='https://tailus.io/sources/blocks/tech-startup/preview/images/clients/microsoft.svg'
          className='h-8 sm:h-10 w-auto lg:h-12'
          alt='microsoft'
        />
      </div>
    </>
  );
};

export default PartnerLogos;
