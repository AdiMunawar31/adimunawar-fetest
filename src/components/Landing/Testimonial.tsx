import React from "react";

const Testimonial = () => {
  return (
    <div>
      <section className='bg-white dark:bg-gray-900 mt-8'>
        <div className='container mx-auto px-6 py-10'>
          <h1 className='text-center text-3xl font-semibold capitalize text-gray-800 dark:text-white lg:text-4xl'>
            What clients saying
          </h1>
          <div className='mx-auto mt-6 flex justify-center'>
            <span className='inline-block h-1 w-40 rounded-full bg-cyan-500'>
              <span className='mx-1 inline-block h-1 w-3 rounded-full bg-cyan-500'>
                <span className='inline-block h-1 w-1 rounded-full bg-cyan-500'></span>
              </span>
            </span>
          </div>
          <div className='mx-auto mt-16 flex max-w-6xl items-start'>
            <button className='hidden rounded-full border p-2 text-gray-800 transition-colors duration-300 hover:bg-gray-100 rtl:-scale-x-100 dark:border-gray-700 dark:text-gray-200 dark:hover:bg-gray-800 lg:block'>
              <svg
                xmlns='http://www.w3.org/2000/svg'
                className='h-6 w-6'
                fill='none'
                viewBox='0 0 24 24'
                stroke='currentColor'
                strokeWidth='{2}'
              >
                <path
                  strokeLinecap='round'
                  strokeLinejoin='round'
                  d='M15 19l-7-7 7-7'
                />
              </svg>
            </button>
            <div>
              <p className='flex items-center text-center text-gray-500 lg:mx-8'>
                Using DiContact management app has been nothing short of a
                game-changer. As a professional constantly on the move, staying
                in touch with my network used to be a daunting task. However,
                this app has transformed my approach to communication and
                organization.
              </p>
              <div className='mt-8 flex flex-col items-center justify-center'>
                <img
                  className='h-14 w-14 rounded-full object-cover'
                  src='https://media.licdn.com/dms/image/D5603AQEe6hXLkIugFQ/profile-displayphoto-shrink_100_100/0/1697479650298?e=1703116800&v=beta&t=91K60lvzDIufZSO5RQHdT8RW6AKya1goGMQ80_-Lh7M'
                  alt=''
                />
                <div className='mt-4 text-center'>
                  <h1 className='font-semibold text-gray-800 dark:text-white'>
                    Adi Munawar
                  </h1>
                  <span className='text-sm text-gray-500 dark:text-gray-400'>
                    Software Dev
                  </span>
                </div>
              </div>
            </div>
            <button className='hidden rounded-full border p-2 text-gray-800 transition-colors duration-300 hover:bg-gray-100 rtl:-scale-x-100 dark:border-gray-700 dark:text-gray-200 dark:hover:bg-gray-800 lg:block'>
              <svg
                xmlns='http://www.w3.org/2000/svg'
                className='h-6 w-6'
                fill='none'
                viewBox='0 0 24 24'
                stroke='currentColor'
                strokeWidth='{2}'
              >
                <path
                  strokeLinecap='round'
                  strokeLinejoin='round'
                  d='M9 5l7 7-7 7'
                />
              </svg>
            </button>
          </div>
        </div>
      </section>
    </div>
  );
};

export default Testimonial;
