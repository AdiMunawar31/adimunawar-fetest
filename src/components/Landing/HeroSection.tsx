import React from "react";
import FeatureIcons from "./FeatureIcons";
import PartnerLogos from "./PartnerLogos";

const HeroSection = () => {
  return (
    <div className='bg-white relative pt-40 pb-20 lg:pt-44 dark:bg-gray-900'>
      <div className='relative xl:container m-auto px-6 md:px-12 lg:px-6'>
        <h1 className='sm:mx-auto sm:w-10/12 md:w-2/3 font-black text-blue-900 text-4xl text-center sm:text-5xl md:text-6xl lg:w-auto lg:text-left xl:text-7xl dark:text-white'>
          Connecting Lives, <br className='lg:block hidden' />{" "}
          <span className='relative text-transparent bg-clip-text bg-gradient-to-r from-blue-600 to-cyan-500 dark:from-blue-400 dark:to-cyan-300'>
            One Contact at a Time.
          </span>
          .
        </h1>
        <div className='lg:flex'>
          <div className='relative mt-8 md:mt-16 space-y-8 sm:w-10/12 md:w-2/3 lg:ml-0 sm:mx-auto text-center lg:text-left lg:mr-auto lg:w-7/12'>
            <p className='sm:text-lg text-gray-700 dark:text-gray-300 lg:w-11/12'>
              Our innovative app redefines the way you connect with your
              contacts, transforming it into a delightful breeze. Say hello to a
              world where communication flows effortlessly, and organization
              becomes a second nature. With our intuitive and powerful tools,
              you'll experience a new level of ease in managing and maintaining
              your relationships
            </p>
            <span className='block font-semibold text-gray-500 dark:text-gray-400'>
              The best companion bot for your chat app.
            </span>
            <FeatureIcons />
            <PartnerLogos />
          </div>
          <div className='mt-12 md:-mt-12 lg:absolute -right-5 lg:w-6/12'>
            <div className='relative w-full'>
              <div
                aria-hidden='true'
                className='absolute scale-75 md:scale-110 inset-0 m-auto w-full h-full md:w-96 md:h-96 rounded-full rotate-45 bg-gradient-to-r from-sky-500 to-cyan-300 blur-3xl'
              />
              <img
                src='https://tailus.io/sources/blocks/tech-startup/preview/images/globalization-cuate.svg'
                className='relative w-full'
                alt='wath illustration'
                loading='lazy'
                width={220}
                height={180}
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};

export default HeroSection;
