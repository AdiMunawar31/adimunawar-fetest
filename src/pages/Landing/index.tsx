import React from "react";
import Footer from "../../components/Footer";
import HeroSection from "../../components/Landing/HeroSection";
import Parallax from "../../components/Landing/Parallax";
import Testimonial from "../../components/Landing/Testimonial";
import Navbar from "../../components/Navbar";

const Landing = () => {
  return (
    <div className='bg-white'>
      <Navbar />
      <HeroSection />
      <Parallax />
      <Testimonial />
      <Footer />
    </div>
  );
};

export default Landing;
