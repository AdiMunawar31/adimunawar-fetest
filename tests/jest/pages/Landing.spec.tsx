import React from "react";
import renderer from "react-test-renderer";
import Landing from "@pages/Landing";
import { MemoryRouter } from "react-router-dom";

describe("Landing component", () => {
  it("renders the Landing component", () => {
    const component = renderer.create(
      <MemoryRouter initialEntries={["/"]}>
        <Landing />
      </MemoryRouter>
    );

    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
