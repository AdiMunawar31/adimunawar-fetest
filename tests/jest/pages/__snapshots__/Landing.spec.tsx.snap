// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing component renders the Landing component 1`] = `
<div
  className="bg-white"
>
  <header>
    <nav
      className="fixed z-20 w-full bg-white/90 dark:bg-gray-900/80 backdrop-blur navbar shadow-2xl shadow-gray-600/5 border-b border-gray-100 dark:border-gray-800 peer-checked:navbar-active dark:shadow-none"
    >
      <div
        className="xl:container m-auto px-6 md:px-12 lg:px-6"
      >
        <div
          className="flex flex-wrap items-center justify-between gap-6 md:py-3 md:gap-0 lg:py-5"
        >
          <div
            className="w-full items-center flex justify-between lg:w-auto"
          >
            <a
              aria-label="logo"
              href="/"
              onClick={[Function]}
            >
              <div
                className="flex"
              >
                <img
                  alt="logo"
                  src="/contact.png"
                  width={40}
                />
                <span
                  className="text-lg text-cyan-400 font-bold ml-2 mt-2"
                >
                  DiContact
                </span>
              </div>
            </a>
            <button
              className="peer-checked:hamburger block relative z-20 p-6 -mr-6 cursor-pointer sm:hidden"
              onClick={[Function]}
            >
              <div
                aria-hidden="true"
                className="m-auto h-0.5 w-5 rounded bg-gray-900 dark:bg-gray-300 transition duration-300"
              />
              <div
                aria-hidden="true"
                className="m-auto mt-2 h-0.5 w-5 rounded bg-gray-900 dark:bg-gray-300 transition duration-300"
              />
            </button>
          </div>
          <div
            className="navmenu hidden w-full flex-wrap justify-end items-center mb-16 space-y-8 p-6 border border-gray-100 rounded-3xl shadow-2xl shadow-gray-300/20 bg-white sm:space-y-0 sm:p-0 sm:m-0 sm:flex sm:flex-nowrap sm:bg-transparent sm:w-10/12 sm:shadow-none sm:border-0"
          >
            <div
              className="text-gray-600 sm:pr-4"
            >
              <ul
                className="space-y-6 tracking-wide font-medium text-base sm:text-sm sm:flex sm:space-y-0"
              >
                <li>
                  <a
                    className="block md:px-4 transition hover:text-primary"
                    href="/"
                    onClick={[Function]}
                  >
                    <span
                      aria-label="Home"
                    >
                      Home
                    </span>
                  </a>
                </li>
                <li>
                  <a
                    className="block md:px-4 transition hover:text-primary"
                    href="/about"
                    onClick={[Function]}
                  >
                    <span
                      aria-label="About"
                    >
                      About
                    </span>
                  </a>
                </li>
              </ul>
            </div>
            <div
              className="w-full space-y-2 border-primary/10 flex flex-col ml-1 sm:flex-row lg:space-y-0 md:w-max lg:border-l"
            >
              <a
                aria-label="Dashboard"
                className="relative flex h-9 -ml-2 sm:ml-8 items-center justify-center sm:px-6 before:absolute before:inset-0 before:rounded-full before:bg-blue-900 before:transition before:duration-300 hover:before:scale-105 active:duration-75 active:before:scale-95"
                href="/dashboard"
                onClick={[Function]}
              >
                <span
                  aria-label="Dashboard"
                  className="relative text-sm font-semibold text-white dark:text-gray-900"
                >
                  Dashboard
                </span>
              </a>
            </div>
          </div>
        </div>
      </div>
    </nav>
  </header>
  <div
    className="bg-white relative pt-40 pb-20 lg:pt-44 dark:bg-gray-900"
  >
    <div
      className="relative xl:container m-auto px-6 md:px-12 lg:px-6"
    >
      <h1
        className="sm:mx-auto sm:w-10/12 md:w-2/3 font-black text-blue-900 text-4xl text-center sm:text-5xl md:text-6xl lg:w-auto lg:text-left xl:text-7xl dark:text-white"
      >
        Connecting Lives, 
        <br
          className="lg:block hidden"
        />
         
        <span
          className="relative text-transparent bg-clip-text bg-gradient-to-r from-blue-600 to-cyan-500 dark:from-blue-400 dark:to-cyan-300"
        >
          One Contact at a Time.
        </span>
        .
      </h1>
      <div
        className="lg:flex"
      >
        <div
          className="relative mt-8 md:mt-16 space-y-8 sm:w-10/12 md:w-2/3 lg:ml-0 sm:mx-auto text-center lg:text-left lg:mr-auto lg:w-7/12"
        >
          <p
            className="sm:text-lg text-gray-700 dark:text-gray-300 lg:w-11/12"
          >
            Our innovative app redefines the way you connect with your contacts, transforming it into a delightful breeze. Say hello to a world where communication flows effortlessly, and organization becomes a second nature. With our intuitive and powerful tools, you'll experience a new level of ease in managing and maintaining your relationships
          </p>
          <span
            className="block font-semibold text-gray-500 dark:text-gray-400"
          >
            The best companion bot for your chat app.
          </span>
          <div
            className="grid grid-cols-3 space-x-4 md:space-x-6 md:flex md:justify-center lg:justify-start"
          >
            <a
              aria-label="add to slack"
              className="p-4 border border-gray-200 dark:bg-gray-800 dark:border-gray-700 rounded-full duration-300 hover:border-cyan-400 hover:shadow-lg hover:shadow-cyan-600/20 dark:hover:border-cyan-300/30"
              href="#"
            >
              <div
                className="flex justify-center space-x-4"
              >
                <img
                  alt="slack logo"
                  className="w-6 h-6"
                  height={128}
                  loading="lazy"
                  src="https://tailus.io/sources/blocks/tech-startup/preview/images/slack.png"
                  width={128}
                />
                <span
                  className="hidden font-medium md:block dark:text-white"
                >
                  Slack
                </span>
              </div>
            </a>
            <a
              aria-label="add to chat"
              className="p-4 border border-gray-200 dark:bg-gray-800 dark:border-gray-700 rounded-full duration-300 hover:border-green-400 hover:shadow-lg hover:shadow-lime-600/20 dark:hover:border-green-300/30"
              href="#"
            >
              <div
                className="flex justify-center space-x-4"
              >
                <img
                  alt="chat logo"
                  className="w-6 h-6"
                  height={128}
                  loading="lazy"
                  src="https://tailus.io/sources/blocks/tech-startup/preview/images/chat.png"
                  width={128}
                />
                <span
                  className="hidden font-medium md:block dark:text-white"
                >
                  Google Chat
                </span>
              </div>
            </a>
            <a
              aria-label="add to zoom"
              className="p-4 border border-gray-200 dark:bg-gray-800 dark:border-gray-700 rounded-full duration-300 hover:border-blue-400 hover:shadow-lg hover:shadow-blue-600/20 dark:hover:border-blue-300/30"
              href="#"
            >
              <div
                className="flex justify-center space-x-4"
              >
                <img
                  alt="chat logo"
                  className="w-6 h-6"
                  height={128}
                  loading="lazy"
                  src="https://tailus.io/sources/blocks/tech-startup/preview/images/zoom.png"
                  width={128}
                />
                <span
                  className="hidden font-medium md:block dark:text-white"
                >
                  Zoom
                </span>
              </div>
            </a>
          </div>
          <div
            className="text-gray-600 font-bold"
          >
            <span>
              Our Partnership :
            </span>
          </div>
          <div
            className="flex gap-6 lg:gap-12 justify-between grayscale lg:w-2/3"
          >
            <img
              alt="airbnb"
              className="h-8 sm:h-10 w-auto lg:h-12"
              src="https://tailus.io/sources/blocks/tech-startup/preview/images/clients/airbnb.svg"
            />
            <img
              alt="ge"
              className="h-8 sm:h-10 w-auto lg:h-12"
              src="https://tailus.io/sources/blocks/tech-startup/preview/images/clients/ge.svg"
            />
            <img
              alt="coty"
              className="h-8 sm:h-10 w-auto lg:h-12"
              src="https://tailus.io/sources/blocks/tech-startup/preview/images/clients/coty.svg"
            />
            <img
              alt="microsoft"
              className="h-8 sm:h-10 w-auto lg:h-12"
              src="https://tailus.io/sources/blocks/tech-startup/preview/images/clients/microsoft.svg"
            />
          </div>
        </div>
        <div
          className="mt-12 md:-mt-12 lg:absolute -right-5 lg:w-6/12"
        >
          <div
            className="relative w-full"
          >
            <div
              aria-hidden="true"
              className="absolute scale-75 md:scale-110 inset-0 m-auto w-full h-full md:w-96 md:h-96 rounded-full rotate-45 bg-gradient-to-r from-sky-500 to-cyan-300 blur-3xl"
            />
            <img
              alt="wath illustration"
              className="relative w-full"
              height={180}
              loading="lazy"
              src="https://tailus.io/sources/blocks/tech-startup/preview/images/globalization-cuate.svg"
              width={220}
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <section
    className="w-full h-[500px] bg-cover bg-fixed bg-center flex flex-col justify-center items-center"
    style={
      {
        "backgroundImage": "url(https://colibriwp.com/blog/wp-content/uploads/2022/02/15.jpg)",
      }
    }
  >
    <div
      className="text-gray-900 text-3xl sm:text-4xl font-semibold text-center mt-40 sm:mt-24 mb-6"
    >
      Stay Up-To-Date
    </div>
    <div
      className="text-center font-bold sm:mt-10 mb-40 sm:mb-28 text-gray-900 max-w-2xl mx-auto p-4"
    >
      "Empowering you with real-time insights, our app is designed to keep your contact information up-to-date. Our automated scans tirelessly work behind the scenes, ensuring that you're always equipped with the latest and most accurate details. Say goodbye to the frustration of outdated contacts and hello to a world where staying informed is effortless. With us, you'll never miss a beat, and you'll always be in control of the most current information. Welcome to a future of seamless, always-accurate connections."
    </div>
  </section>
  <div>
    <section
      className="bg-white dark:bg-gray-900 mt-8"
    >
      <div
        className="container mx-auto px-6 py-10"
      >
        <h1
          className="text-center text-3xl font-semibold capitalize text-gray-800 dark:text-white lg:text-4xl"
        >
          What clients saying
        </h1>
        <div
          className="mx-auto mt-6 flex justify-center"
        >
          <span
            className="inline-block h-1 w-40 rounded-full bg-cyan-500"
          >
            <span
              className="mx-1 inline-block h-1 w-3 rounded-full bg-cyan-500"
            >
              <span
                className="inline-block h-1 w-1 rounded-full bg-cyan-500"
              />
            </span>
          </span>
        </div>
        <div
          className="mx-auto mt-16 flex max-w-6xl items-start"
        >
          <button
            className="hidden rounded-full border p-2 text-gray-800 transition-colors duration-300 hover:bg-gray-100 rtl:-scale-x-100 dark:border-gray-700 dark:text-gray-200 dark:hover:bg-gray-800 lg:block"
          >
            <svg
              className="h-6 w-6"
              fill="none"
              stroke="currentColor"
              strokeWidth="{2}"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M15 19l-7-7 7-7"
                strokeLinecap="round"
                strokeLinejoin="round"
              />
            </svg>
          </button>
          <div>
            <p
              className="flex items-center text-center text-gray-500 lg:mx-8"
            >
              Using DiContact management app has been nothing short of a game-changer. As a professional constantly on the move, staying in touch with my network used to be a daunting task. However, this app has transformed my approach to communication and organization.
            </p>
            <div
              className="mt-8 flex flex-col items-center justify-center"
            >
              <img
                alt=""
                className="h-14 w-14 rounded-full object-cover"
                src="https://media.licdn.com/dms/image/D5603AQEe6hXLkIugFQ/profile-displayphoto-shrink_100_100/0/1697479650298?e=1703116800&v=beta&t=91K60lvzDIufZSO5RQHdT8RW6AKya1goGMQ80_-Lh7M"
              />
              <div
                className="mt-4 text-center"
              >
                <h1
                  className="font-semibold text-gray-800 dark:text-white"
                >
                  Adi Munawar
                </h1>
                <span
                  className="text-sm text-gray-500 dark:text-gray-400"
                >
                  Software Dev
                </span>
              </div>
            </div>
          </div>
          <button
            className="hidden rounded-full border p-2 text-gray-800 transition-colors duration-300 hover:bg-gray-100 rtl:-scale-x-100 dark:border-gray-700 dark:text-gray-200 dark:hover:bg-gray-800 lg:block"
          >
            <svg
              className="h-6 w-6"
              fill="none"
              stroke="currentColor"
              strokeWidth="{2}"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M9 5l7 7-7 7"
                strokeLinecap="round"
                strokeLinejoin="round"
              />
            </svg>
          </button>
        </div>
      </div>
    </section>
  </div>
  <footer
    className="bg-white dark:bg-gray-900"
  >
    <div
      className="container mx-auto px-6 py-12 mt-20"
    >
      <div
        className="md:-mx-3 md:flex md:items-center md:justify-between"
      >
        <h1
          className="text-3xl font-semibold tracking-tight text-gray-800 dark:text-white md:mx-3 xl:text-4xl"
        >
          Subscribe our newsletter to get update.
        </h1>
        <div
          className="mt-6 shrink-0 md:mx-3 md:mt-0 md:w-auto"
        >
          <a
            className="inline-flex w-full items-center justify-center rounded-lg bg-blue-900 px-4 py-2 text-sm text-white duration-300 hover:bg-gray-700 focus:ring focus:ring-gray-300 focus:ring-opacity-80"
            href="/subsribe-dicontact"
          >
            <span
              className="mx-2"
            >
              Subscribe Now
            </span>
            <svg
              className="mx-2 h-6 w-6"
              fill="none"
              stroke="currentColor"
              strokeWidth="1.5"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M17.25 8.25L21 12m0 0l-3.75 3.75M21 12H3"
                strokeLinecap="round"
                strokeLinejoin="round"
              />
            </svg>
          </a>
        </div>
      </div>
      <hr
        className="my-6 border-gray-200 dark:border-gray-700 md:my-10"
      />
      <div
        className="flex flex-col items-center justify-between sm:flex-row"
      >
        <a
          className="text-2xl font-bold text-gray-800 transition-colors duration-300 hover:text-gray-700 dark:text-white dark:hover:text-gray-300"
          href="/"
          onClick={[Function]}
        >
          <div
            className="flex"
          >
            <img
              alt="logo"
              src="/contact.png"
              width={40}
            />
            <span
              className="text-lg text-cyan-400 font-bold ml-2 mt-2"
              id="DiContact"
            >
              DiContact
            </span>
          </div>
        </a>
        <p
          className="mt-4 text-sm text-gray-500 dark:text-gray-300 sm:mt-0"
        >
          © Copyright 2023. All Rights Reserved.
        </p>
      </div>
    </div>
  </footer>
</div>
`;
